okta/UsersClient.py

```diff
@@ -49,18 +49,24 @@ def update_user(self, user):
         :type user: User
         :rtype: User
         """
+
         return self.update_user_by_id(user.id, user)
 
     def update_user_by_id(self, uid, user):
         """Update a user, defined by an id
 
         :param uid: the target user id
         :type uid: str
-        :param user: the data to update the target user
+        :param params: the data to update the target user
```

Comment:
This should match the argument, which should remain as user. Otherwise, there'd be no need to parse it out below.